modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+.Procedure
+Use the following procedure to deploy {product-title} for vSphere storage on different zones and regions.
+
+. In the VMware vCenter vSphere client GUI:
+.. Create and assign `openshift-zone` and `openshift-region` tags as needed to the hosts on the desired clusters.
+.. Create and assign `cat-zone-local-datastore` tags to the desired datastores.
+
++
+For more information about vSphere tags, see the WMware vSphere documentation.
```

Comment:
I think we should add a small example table for a 6 nodes clusters, 3 masters / 3 worker spread on 3 zones. Show what category / tag the customer should use.

Reply:
Done.